server/test/utils.ts

```diff
@@ -50,7 +40,7 @@ async function createUser({
   // build object
```

Comment:
on the line above this, we should just be using `UserObject` instead of `Partial<UserObject>` since this function is expected to return values identical to the types we're expecting in the user model